### DISCUSSION TITLE
Number of Parallel Jobs

### DISCUSSION THREAD

**Discussion by reikdas** [community/community#26422] [Actions]
<p>Is there a limit to the number of parallel jobs that can be triggered at the same time? The description of “Jobs” here - <a href="https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions</a> seems to indicate that there could be unlimited jobs triggered in parallel.</p>

**Comment by airtower-luna** [comment]
<p>The “usage limits” documentation mentions that the maximum number of parallel jobs depends on your plan:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits" target="_blank" rel="noopener">Usage limits, billing, and administration - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Additional jobs will just have to wait until spots free up.</p>

**Comment by reikdas** [comment]
<p>So the number of parallel macOS jobs is limited to 5, but the number of jobs is unlimited for Linux and Windows?</p>

**Comment by airtower-luna** [comment]
<p>No, check the “Total concurrent jobs” column of the table. That’s what applies to Linux and Windows.</p>

**Comment by reikdas** [comment]
<p>Thanks!</p>
<p>And if I also have macOS jobs, is the total number of concurrent jobs 25 for the free tier (20 for any combination of Linux and Windows; 5 for macOS)?<br>
Or is the total number 20 and I can have upto 5 macOS jobs among those 20?</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="reikdas" data-post="5" data-topic="140788">
<div class="title">
<div class="quote-controls"></div>
 reikdas:</div>
<blockquote>
<p>Or is the total number 20 and I can have upto 5 macOS jobs among those 20?</p>
</blockquote>
</aside>
<p>That’s how I understand the table, but if you need a definitive answer you’ll have to ask someone from Github. :wink:</p>
<p>That said, it’s not like trying to start more concurrent jobs will break anything, it’s just that some of them will have to wait until spots free up.</p>
